src/expr/src/scalar/func.rs

```diff
@@ -3214,6 +3272,14 @@ impl UnaryFunc {
             ListLength => ScalarType::Int64.nullable(true),
 
             RegexpMatch(_) => ScalarType::Array(Box::new(ScalarType::String)).nullable(true),
+
+            Cos => ScalarType::Float64.nullable(true),
+            Cosh => ScalarType::Float64.nullable(true),
+            Sin => ScalarType::Float64.nullable(true),
+            Sinh => ScalarType::Float64.nullable(true),
+            Tan => ScalarType::Float64.nullable(true),
+            Tanh => ScalarType::Float64.nullable(true),
+            Cot => ScalarType::Float64.nullable(true),
```

Comment:
I think these functions can't actually return `NULL` unless their input is null. If that's true, then `s/true/in_nullable/`.